### ISSUE TITLE
[1.10.2] Routed Request Pipe losing items

### ISSUE THREAD

**Issue by sareni19** [MrTJP/ProjectRed#1236]
When requesting an item from my chests, I lose some of the item requested. For example, I had 32 gold ingots, and when I requested all of them, I only got 18. The rest are nowhere to be found and do not pull up in the request pipe. Any idea on what could be causing this? Not sure if it will matter, but I am using FTB Beyond 1.11.0.

**Comment by Psy-Virus** [comment]
Hey there,
if you use the latest version of Project Red, you're maybe affected by [this issue](#1187 ).

When the items get stuck you also get wrong counts in the network/request pipe.
For example, we had about 7k cobble stone. Then something got stuck and it showed more than twice as much when we requested it, until we fixed the stuck items.
Mostly happens, when the server is freshly started and not all chunks are loaded (I guess).
But also happens sometimes after a few hours.. So it's not easy producible.
